components/molecules/CardHorizontalBarChart/card-horizontal-bar-chart.tsx

```diff
@@ -19,22 +17,15 @@ export interface LanguageObject {
 
 interface CardHorizontalBarChartProps {
   languageList: LanguageObject[];
-  withDescription: boolean;
 }
 
 const languageToColor: AllSimpleColors = colors as AllSimpleColors;
 
-const CardHorizontalBarChart = ({ languageList, withDescription }: CardHorizontalBarChartProps): JSX.Element => {
```

Comment:
I removed the `withDescription` prop because it was only used in one spot, the `<CardHorizontalBarChart withDescription={false} languageList={languageList} />`, but it was set to `false` always.